spidermon/core/actions.py

```diff
@@ -37,6 +38,8 @@ def run(self, result, data):
             result.add_action_skip(self, e.args[0])
         except:
             result.add_action_error(self, traceback.format_exc())
+            if self.fallback is not None:
```

Comment:
**issue** We need tests for this fallback action.

Reply:
Tests added now!